pkg/athenz/model.go

```diff
@@ -92,7 +110,7 @@ func getMembersForRole(domain *zms.DomainData) RoleMembers {
 }
 
 // ConvertAthenzPoliciesIntoRbacModel transforms the given Athenz Domain structure into role-centric policies and members
-func ConvertAthenzPoliciesIntoRbacModel(domain *zms.DomainData) Model {
+func ConvertAthenzPoliciesIntoRbacModel(domain *zms.DomainData, crCache *cache.SharedIndexInformer) Model {
```

Comment:
Can we add a unit test which actually checks the Model object which is returned from this function? We can populate the fake informer with a trust domain object.